homeassistant/components/growatt_server/sensor.py

```diff
@@ -83,6 +83,10 @@ async def async_setup_entry(
     api = growattServer.GrowattApi()
     api.server_url = url
 
+    # Temporary work-around for #81951 (https://github.com/home-assistant/core/issues/81951) - Will be addressed correctly in a subsequent PR
+    headers = {"User-Agent": "Dalvik/2.1.0 (Linux; U; Android Device)"}
```

Comment:
We don't allow this kind of data in integrations. Not even temporarily. It should be in the library.

Reply:
ok, no worries, thanks @MartinHjelmare - I'll submit a change upstream and wait for it to be released and then re-submit this as a version bump.